font/src/ft/mod.rs

```diff
@@ -294,6 +294,29 @@ impl FreeTypeRasterizer {
 
         let (pixel_width, buf) = Self::normalize_buffer(&glyph.bitmap())?;
 
+        // Render a custom symbol for the underline cursor
+        if glyph_key.c == '􊏢' {
```

Comment:
Since we're using specific unicode area, it might be more clear to use `\u{ffff}` notation. Additionally, we should probably define a constant for this somewhere.